gravitee-am-ui/src/app/domain/applications/application/advanced/factors/info-banner/info-banner.component.html

```diff
@@ -16,11 +16,15 @@
 
 -->
 
-<div *ngIf="showInfo" fxLayout="row" fxLayoutAlign="center center" class="show-info" [style]="getColors()">
-  <div fxFlex="5" style="padding: 10px"><mat-icon>info_outline</mat-icon></div>
-  <div fxFlex="65" style="gap: 4px">{{ text }}</div>
-  <button fxFlex="25" (click)="gotoLink($event)" mat-raised-button class="more-info-button">
-    {{ buttonName }}
-  </button>
-  <div fxFlex="5" fxLayoutAlign="end center" class="close-info"><mat-icon (click)="closeInfo()">close</mat-icon></div>
+<div *ngIf="showInfo" fxLayout="row" fxLayoutAlign="start center" class="show-info" [style]="getColors()">
```

Comment:
We could add here min-width: 320px; to keep the background on all elements in smaller screens. What do you think?

Reply:
but it doesnt change anything. On smaller devices than 320px the view is entirely not usable